test/unit/src/signals/SignalsTest.cpp

```diff
@@ -102,8 +102,9 @@ TEST_CASE( "signals/Signals" )
 	{
 		class MemberClass {
 		public:
+			MemberClass() { mAccum = 0; }
 			void handler()	{ mAccum++;	}
-			int mAccum = 0;
```

Comment:
Sadly, it's not even a VS issue, it's specific to VS 2013. The original definition with the default initializer builds just fine in 2015 :(